Katello/katello
# Fixes #26996 - support docker content

To test this out:

- Create a new repository:
    - type: docker
    - upstream url: https://registry.hub.docker.com/
    - upstream repository name: library/busybox
- Click 'save'
- Go to the new docker repository
- Ensure that settings can be edited without error
    - Especially ensure that Limit Sync Tags can be emptied after being set to something.
- Finally, try deleting the repository
